### PR TITLE
Run Tests in parallel

### PR DESCRIPTION
**Pull request by odinuv**
Run AWS and Azure tests in parallel, since they're running in independent projects.

Part of https://keboola.atlassian.net/browse/AIS-35
Depedns on https://github.com/keboola/sapi-python-client/pull/71

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -14,57 +13,102 @@ env:
...
+        uses: actions/checkout@v3
+
+      - name: Download image
+        uses: ishworkh/docker-image-artifact-download@v1
+        with:
+          image: "sapi-python-client"
+
+      - name: Run Tests
+        run: |
+          docker-compose run --rm -e KBC_TEST_TOKEN=$KBC_AZ_TEST_TOKEN -e KBC_TEST_API_URL=$KBC_AZ_TEST_API_URL -e SKIP_ABS_TESTS=1 sapi-python-client -m unittest
```
Comment (pivnicek) [inline .github/workflows/push.yml:87]: Do we really want `SKIP_ABS_TESTS=1` for `tests_azure`?

Maybe we can use a condition? `if: !$SKIP_ABS_TESTS` or stg like that?

Comment (odinuv) [reply .github/workflows/push.yml:87]: No we don't want to skip them :)

Not sure how the conditon would work

Comment (odinuv) [reply .github/workflows/push.yml]: Fixed

Comment (odinuv) [reply .github/workflows/push.yml]: Ok, now it is really fixed

Comment (odinuv) [reply .github/workflows/push.yml]: No skipped tests on Azure 

image

**Pull request merged**